package.json

```diff
@@ -18,7 +18,7 @@
 	"types": "./source/index.d.ts",
 	"sideEffects": false,
 	"engines": {
-		"node": "^12.17.0 || ^14.13 || >=16.0.0"
+		"node": ">=18"
```

Comment:
We don't plan to do a major release for Node.js 18 support. It's not worth having to do a major release.

Reply:
Oops, you're totally right! I have simplified my approach in #642, which supersedes this PR